test/browser/webgl_offscreen_framebuffer_swap_with_bad_state.c

```diff
@@ -28,7 +28,7 @@ int main() {
   EMSCRIPTEN_WEBGL_CONTEXT_HANDLE ctx = emscripten_webgl_create_context("#canvas", &attr);
   emscripten_webgl_make_context_current(ctx);
 
-#if !TEST_WEBGL2 && TEST_VAO
+#if !TEST_WEBGL2 && TEST_REQUIRE_VAO
   // This test cannot run without browser support for OES_vertex_array_object.
```

Comment:
This is not the ideal state but it documents what I believe to be the current reality.

```suggestion
  // This test cannot run without browser support for OES_vertex_array_object.
  // This check is just to verify that the browser has support; otherwise, we
  // will end up testing the non-VAO path. Enabling it here does not actually do
  // anything, because offscreen framebuffer has already been initialized. Note
  // that if GL_SUPPORT_AUTOMATIC_ENABLE_EXTENSIONS=0, then offscreen
  // framebuffer will _never_ use VAOs on WebGL 1 (unless something enables
  // OES_vertex_array_object before createOffscreenFramebuffer runs).
```